packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-drilldown.vue

```diff
@@ -574,16 +584,15 @@ const initialize = async () => {
 	const modelConfigurationId = props.node.inputs[0].value?.[0];
 	if (!modelConfigurationId) return;
 	modelConfiguration.value = await getModelConfigurationById(modelConfigurationId);
-	const model = await getAsConfiguredModel(modelConfiguration.value);
+	model.value = await getAsConfiguredModel(modelConfiguration.value);
```

Comment:
Can we add a FIXME here? It seems like we are getting model information, per @dvince2 this is not the right way to do it even though it may look like it is working currently.

Reply:
I added the comment.